src/net/udp.rs

```diff
@@ -198,15 +208,25 @@ impl UdpSocket {
         self.inner.connect(SockAddr::from(socket_addr)).await
     }
 
-    /// Sends data on the socket to the given address.
+    /// Sends data on the connected socket
+    ///
+    /// On success, returns the number of bytes written.
+    pub async fn send<T: BoundedBuf>(
+        &self,
+        buf: T,
+    ) -> crate::BufResult<usize, T> {
+        self.inner.send_to(buf, None).await
+    }
+
+    /// Sends data on the socket to the given address or to previously connected remote peer.
```

Comment:
Since your comment here says `or to a previously connected remote peer`, want to add that `send` might be preferred in the case of a previously connected remote peer as it saves a box?

Reply:
I removed comment about connected sockets, because `UdpSocket::send_to`can't be used without SocketAdddr now